src/pages/api/__tests__/crawl.test.ts

```
@@ -0,0 +1,92 @@
+// Mock the crawler module
+jest.mock('@/lib/crawler', () => ({
+  crawlEthGlobalShowcase: jest.fn(),
+}));
+
+const mockCrawlEthGlobalShowcase = crawlEthGlobalShowcase as jest.MockedFunction<typeof crawlEthGlobalShowcase>;
+
+describe('/api/crawl', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    // Reset environment variable
+    delete process.env.NEXT_PUBLIC_ENVIRONMENT;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)